### PR TITLE
web scraping example

### PR DESCRIPTION
**Pull request by dleviminzi**

### CODE REVIEW

**Review by mernit — CHANGES_REQUESTED** [review]
we should use our .map() here, the idea is for it to replace thread pool executor

**Comment by dleviminzi** [comment]
> we should use our .map() here, the idea is for it to replace thread pool executor

Map requires you to know the number of things you are hitting ahead of time. For instance if we had a list of urls we wanted to hit already that would work fine. However, in this example we are crawling and therefore the inputs are not known ahead of time.

File: web_scraping/README.md [hunk 1]
```diff
@@ -2,4 +2,14 @@
 
 The code in this example demonstrates how Beam functions can be used to scrape Wikipedia pages in parallel.
 
-Run this example: `python app.py`
+## Mapped Crawl
```
Comment (mernit) [inline web_scraping/README.md:5]: title should be in layman's terms

File: web_scraping/README.md [hunk 2]
```diff
@@ -2,4 +2,14 @@
...
+
+The mapped crawl uses the `map` method from the `function` decorator to launch a remote function calls for each element of the input list.
```
Comment (mernit) [inline web_scraping/README.md:7]: there is a more descriptive way of explaining this than `mapped crawl`

File: web_scraping/README.md [hunk 3]
```diff
@@ -2,4 +2,14 @@
 
 The code in this example demonstrates how Beam functions can be used to scrape Wikipedia pages in parallel.
 
-Run this example: `python app.py`
+## Mapped Crawl
+
+The mapped crawl uses the `map` method from the `function` decorator to launch a remote function calls for each element of the input list. 
+
+Run this example: `python mapped_crawl.py`
```
Comment (mernit) [inline web_scraping/README.md:9]: rename

File: web_scraping/README.md (hunk 1 above)
Comment (mernit) [reply web_scraping/README.md:5]: instead of explaining what it is internally, explain it from a user's perspective i.e. "running a parallelized web scraper"

as a thought experiment, you can look at anything you write and ask, "would a user explain it like this?"

File: web_scraping/README.md [hunk 4]
```diff
@@ -2,4 +2,14 @@
...
+
+## Threaded Crawl
```
Comment (mernit) [inline web_scraping/README.md:11]: rename

**Review by mernit — CHANGES_REQUESTED** [review]

**Review by mernit — APPROVED** [review]
looks good, let's also add this to the docs in the examples section

**Comment by dleviminzi** [comment]
> looks good, let's also add this to the docs in the examples section

Yeah I'm working on the write up rn
